BEIMA.Client/src/pages/Devices/DevicesPage.js

```diff
@@ -22,6 +23,16 @@ const DevicesPage = () => {
     const loadData = async () => {
       setLoading(true)
       let devices = await DeviceListCall();
+      let type;
+      devices = await Promise.all(devices.map(async (item) => {
+        type = await GetDeviceType(item.deviceType)
+        if(type.status === 404){
+          item['deviceTypeName'] = '';
```

Comment:
Should we display 'resource not found' on the page somewhere instead?

Reply:
Updated to display "Device Type Not Found"